examples/external-and-internal-lbs/main.tf

```diff
@@ -12,20 +12,21 @@ module "internal_cert" {
 }
 
 module "internal_lb" {
-  source = "superblocksteam/superblocks/aws//modules/lb"
+  source  = "superblocksteam/superblocks/aws//modules/load-balancer"
```

Comment:
```suggestion
  source  = "superblocksteam/superblocks/aws/modules/load-balancer"
```

Reply:
Submodules are meant to be invoked with double slashes

https://registry.terraform.io/modules/terraform-aws-modules/rds/aws/1.0.3/submodules/db_instance